rules/app.bzl

```diff
@@ -61,9 +61,16 @@ def ios_application(name, apple_library = apple_library, **kwargs):
     application_kwargs["launch_storyboard"] = application_kwargs.pop("launch_storyboard", library.launch_screen_storyboard_name)
     application_kwargs["families"] = application_kwargs.pop("families", ["iphone", "ipad"])
 
+    local_debug_options_for_swift = []
+    if library.has_swift_sources:
+      local_debug_options_for_swift += ["@build_bazel_rules_ios//rules:_LocalDebugOptions"]
```

Comment:
should we update some documentation / readme to reflect this important flag?

Reply:
I added comments in the BUILD.bazel where the _LocalDebugOptions target is defined. I will also reference this README: https://github.com/ios-bazel-users/ios-bazel-users/blob/master/DebuggableRemoteSwift.md

Reply:
Adding separate docs shouldn't be necessary, since the xcodeproj rule automatically enables the flag when necessary